packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -235,9 +245,22 @@ function ShoppingPane(props) {
 
   const handleClearInput = useCallback(() => {
     setSearchTerm('');
-    debouncedProductsQuery('', orderby, order);
+    setPage(1);
+    setLoaded(false);
+    debouncedProductsQuery('', 1, orderby, order);
   }, [debouncedProductsQuery, order, orderby]);
 
+  const onLoadMore = useCallback(() => {
+    if (canLoadMore) {
+      const newPage = page + 1;
+      setPage(newPage);
+      debouncedProductsQuery(searchTerm, newPage, orderby, order);
```

Comment:
Let's use `searchProducts` here instead. Otherwise this will trigger a tracking event, which we don't need for infinite scrolling.